src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -62,30 +61,10 @@ export const roomTitleSchema = Yup.string()
 
 export const validationSchema_v2 = Yup.object()
   .shape<SchemaShape>({
-    name: roomTitleSchema.when(
-      "$editing",
-      (editing: boolean, schema: Yup.StringSchema) =>
-        !editing
-          ? schema
-              .test(
-                "name",
-                "Must have alphanumeric characters",
-                (val: string) => createSlug(val).length > 0
-              )
-              .test(
-                "name",
-                "This venue name is already taken",
-                async (val: string) =>
-                  !val ||
-                  !(
-                    await firebase
-                      .firestore()
-                      .collection("venues")
-                      .doc(createSlug(val))
-                      .get()
-                  ).exists
-              )
-          : schema //will be set from the data from the api. Does not need to be unique
+    name: roomTitleSchema.test(
+      "name",
+      "Must have alphanumeric characters",
+      (val: string) => createSlug(val).length > 0
```

Comment:
Maybe the check here should be on the result of `generateSlug()` so that no matter how many special chars are input, only those who matter are counted

Reply:
I'm just reusing the check from the other forms, we can definitely improve it with your proposal but IMO it should be done around all of the forms at the same time in another PR.

Reply:
@denisdimitrov if that's the case, I see it as a must i.e. create issue/assign/deal with instead of fire and forget.

Invalid checks in this may not only cause user frustration (false negatives) but maintenance headaches later on (false positives).